### DISCUSSION TITLE
Host emoticons in assets.cratch.mit.edu

### DISCUSSION THREAD

**Discussion by ghost** [vercte/forumoji#21] [Polls]

**Comment by Autofirejm** [comment]
very yes, let's host them there

**Reply by ghost** [reply]
Yeah, because cube is blocked for me

**Comment by dinomaster20** [comment]
I'm guessing "emoticons" is just emojis? If so, yes, unless that would cause any other issues/difficulties. I can't even use cube upload as my email won't work, and I think my addon makes it automatically assets.scratch.mit.edu .

**Reply by ghost** [reply]
> I'm guessing "emoticons" is just emojis? 

Correct
